packages/gafl-webapp-service/src/pages/licence-details/licence-for/__tests__/route.spec.js

```diff
@@ -0,0 +1,21 @@
+import { validator } from '../route'
+
+describe('licence-for > route', () => {
+  describe('validator', () => {
+    it('should return an error if licence-for is not you or someone-else', () => {
+      const result = validator.validate({ 'licence-for': 'none' })
+      expect(result.error).not.toBeUndefined()
+      expect(result.error.details[0].message).toBe('"licence-for" must be one of [you, someone-else]')
+    })
+
+    it('should not return an error if licence-for is you', () => {
+      const result = validator.validate({ 'licence-for': 'you' })
+      expect(result.error).toBeUndefined()
+    })
+
+    it('should not return an error if licence-for is someone-else', () => {
+      const result = validator.validate({ 'licence-for': 'someone-else' })
+      expect(result.error).toBeUndefined()
+    })
+  })
+})
```

Comment:
So you're testing Joi here, which is fine, but you're not testing the pageRoute at all. You need to mock that and test the inputs.